packages/db/src/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { MissingRequiredEnvVarError } from '@cumulus/errors';
+
+export const getRequiredEnvVar = (
+  name: string,
+  env: NodeJS.ProcessEnv = process.env
+): string => {
+  const value = env[name];
+
+  if (typeof value === 'string') return value;
+
+  throw new MissingRequiredEnvVarError(`The ${name} environment variable must be set`);
+};
```

Comment:
I'm curious why we need to pull this into here and can't use envUtils

Reply:
Need is too strong a word.    This just demonstrates *a* solution to the lerna dependency issue.  I'm not clear (but will look tomorrow) on exactly what the dependency graph resolution is.    This sort of thing is an ongoing concern with lerna bootstrap dependency resolution and  a complex sub-dependency graph.